### PR TITLE
Community Article - Configuring vhosts on a LAMP Stack

### PR DESCRIPTION
**Pull request by BadCrumpany**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-badd7c9501/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-3180b22783/how-to/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7d8c7219f9/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-0959ca7a1d/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-77b2a80961/how-to/).

**Comment by WilliamLoy** [comment]
@cyrichardson this one is ready for you :)

**Comment by cyrichardson** [comment]
@WilliamLoy On it.

**Comment by cyrichardson** [comment]
@WilliamLoy Suggested changes here: 
[configuring-vhosts-on-a-lamp-stack_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2823599/configuring-vhosts-on-a-lamp-stack_cyrEdits.docx)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-22eb10256b/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-eeec3c7f59/how-to/).

File: content/cloud-servers/configuring-vhosts-on-a-lamp-stack.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    `$ mkdir -p /var/www/vhosts/yoursitename.com`
+
+    **Note:** The vhost that you made previously is configured to look in this directory.
+
+5. Repeat steps 1 - 4 to add additional vhosts.
+
+
+### Test your vhost configuration
+
+1. In this step you should see a response of `Syntax OK`. Test the configuration by using the following command:
```
Comment (RackerKate) [inline content/cloud-servers/configuring-vhosts-on-a-lamp-stack.md]: Suggest either adding an intro sentence for the steps, or possibly removing the numbering for this section. It's only 2 steps anyway.

**Pull request merged**